anvio/__init__.py

```diff
@@ -844,6 +844,31 @@ def get_args(parser):
                       filtering criteria. In other programs, you may get everything, nothing, or an error. It really depends\
                       on the situation. Fortunately, mistakes are cheap, so it's worth a try."}
                 ),
+    'flank-mode': (
+            ['-f','--flank-mode'],
```

Comment:
it is a good idea to make sure the abbreviated forms of these flags (i.e., `-f`) do not appear in this file. 

because when it is the case, mixing them will cause errors, and there will be lots of headache. if you can't find an abbreaviated form that makes sense, you can elect to not use any and keep it as `['--flank-mode']`.

Reply:
removed abbreviated flag